### ISSUE TITLE
Showing Link Size (Edge Weight)

### ISSUE THREAD

**Issue by efefurkankarakaya** [vasturiano/react-force-graph#405]
**Is your feature request related to a problem? Please describe.**
It's not a problem but if it can be implemented somehow or there'll be prop or function that provides adding edge weights would be great.

**Describe the solution you'd like**
I think it can be added by a prop like `showWeights = { true }`.

**Additional context**
![edge](https://user-images.githubusercontent.com/49307777/206782253-d07751ee-b6b7-4591-984a-0bf6ef095e3a.png)

**Comment by vasturiano** [comment]
@efefurkankarakaya you can see the following example for adding text to your links:
https://vasturiano.github.io/react-force-graph/example/text-links/index-3d.html

**Comment by efefurkankarakaya** [comment]
@vasturiano Yes, thanks. I used that but I actually meant, it can be easier.
